src/app/pages/scan/Scan.module.css

```diff
@@ -0,0 +1,31 @@
+.container {
+  display: grid;
+  grid-template-rows: 25% 50% 25%;
+  width: 100%;
+  margin: 0 auto;
+  justify-items: center;
+  align-items: center;
```

Comment:
Maybe you could sum them up into "place-items: center"

Reply:
yes, thanks!